From f3420b143440d5130db7e2ce9479c79bd215c3ac Mon Sep 17 00:00:00 2001
From: Andrew Morton <akpm@linux-foundation.org>
Date: Fri, 28 Oct 2022 15:42:53 -0700
Subject: [PATCH 34/44] 
 maple_tree-reorganize-testing-to-restore-module-testing-fix-2

fix it some more

Cc: Liam Howlett <liam.howlett@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 lib/maple_tree.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index 53f2fb84d..54350221c 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -6018,6 +6018,7 @@ void *mas_find(struct ma_state *mas, unsigned long max)
 	/* Retries on dead nodes handled by mas_next_entry */
 	return mas_next_entry(mas, max);
 }
+EXPORT_SYMBOL_GPL(mas_find);
 
 /**
  * mas_find_rev: On the first call, find the first non-null entry at or below
-- 
2.38.1.385.g3b08839926

